crs_application/src/task_managers/motion_planning_manager.cpp

```diff
@@ -334,13 +334,6 @@ common::ActionResult MotionPlanningManager::planProcessPaths()
       continue;
     }
 
-    if (plan.start.points.empty() || plan.end.points.empty())
-    {
-      RCLCPP_ERROR(node_->get_logger(), "Start or End move for process %i are empty, invalidating plan", i);
-      plan.process_motions.clear();
-      continue;
-    }
```

Comment:
The new method of the motion planning server causes there to be no end motion if it isn't the last process motion, so that check was causing it to fail.